docs/node/tutorial.mdx

```diff
@@ -28,6 +33,9 @@ To create the node's configuration files, a new `genesis.json`, and a new keypai
 kwil-admin setup init -o ./testnet/node0
 ```
 
+The `./testnet/node0` folder will be the "root" directory for `kwild` that
+contains the configuration files, including `genesis.json` and `config.toml`.
+
 ### Running The Node
 
 Using the `kwild` binary, we can begun running the network with this single node:
```

Comment:
Should we give quickstart commands for starting postgres in this tutorial?

Also, does the kwild command need flags for postgres?

Reply:
I also think this comment applies to the setup and start instructions for node 2. It would be great if someone could complete each step of this tutorial and have it "just work".

Reply:
The `kwild` defaults will work with any of our postgres Docker solutions, image or compose.

I'm leaning toward having the tutorial, which is more involved and instructive than the quickstart, to just say start your postgres instance as described in installation.  We can precede the `kwild` command with the docker run command though.  I think that would be fine and it would show clearly where the postgresql data folder is specified.... ok I'll update like that.

Reply:
I *think* this thread is resolved, but a lot has changed in the last two days.  We can replicate commands from other pages here if needed though.